euth_wagtail/templates/base.html

```diff
@@ -55,12 +52,16 @@
         </div>
 
         {% block footer %}
-
+        {% include "includes/footer.html" %}
```

Comment:
Befor the footer was added on each page, because on some pages (where A3 is embedded) the footer should not show